tests/functional/conftest.py

```diff
@@ -7,6 +7,7 @@
 from tests.common.fixtures import es_client  # noqa: F401
 from tests.common.fixtures import init_elasticsearch  # noqa: F401
 from tests.common.fixtures.elasticsearch import ELASTICSEARCH_INDEX, ELASTICSEARCH_URL
+from tests.functional.fixtures.authentication import *  # noqa: F403,F401
```

Comment:
I kind of prefer just keeping all the global and shared fixtures in the `conftest.py` files, to be honest. It's a more obvious location, makes them easier to find. There shouldn't be too many of them anyway. Over-organizing just makes the code more of a pain to navigate